triton/whisper/model_repo_whisper/decoder/config.pbtxt

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "decoder"
+backend: "onnxruntime"
+default_model_filename: "large-v2-decoder-fp16.onnx"
+
+max_batch_size: 16
+input [
+  {
+    name: "tokens"
+    data_type: TYPE_INT64
+    dims: [-1]
+  },
+  {
+    name: "in_n_layer_self_k_cache"
+    data_type: TYPE_FP32
+    dims: [32, -1, 1280]
```

Comment:
Hard-coded values in this file are valid only for large-v2. Please 
use `large-v2` in the filename so that we can support other types of models in the future.